docs/source/gui.rst

```diff
@@ -0,0 +1,18 @@
+Graphic interface
+*****************
+  
+List of functions in the ``gui`` folder: those are mostly to handle the "graphic interface"
+that can be used to deal with ``userInputs()``.
+
+----
+
+.. automodule:: src.gui 
+
+.. autofunction:: askForGroupAndOrSession
+.. autofunction:: askUserCli
```

Comment:
this one for example is not for the GUI, it could be confusing ti have it there

Reply:
Good point.

Reply:
This should be fixed now.